modules/apps/collaboration/upload/upload-web/src/main/java/com/liferay/upload/web/internal/DefaultUploadResponseHandler.java

```diff
@@ -131,4 +145,40 @@ public JSONObject onSuccess(
 		return jsonObject;
 	}
 
+	private String _getAllowedFileExtensions() {
+		String allowedFileExtensionsString = StringPool.BLANK;
+
+		try {
+			DLConfiguration dlConfiguration =
```

Comment:
Instead of using ConfigurationProviderUtil to fetch the configuration we should use ConfigurableUtil in the activate method of the component. See https://github.com/liferay/liferay-portal/blob/master/modules/apps/collaboration/document-library/document-library-service/src/main/java/com/liferay/document/library/internal/util/DLValidatorImpl.java#L249 as an example about how to do that. We shouldn't need the try/catch with that solution.

Reply:
Thanks for that! 😃